resources/views/layout.blade.php

```diff
@@ -55,42 +55,53 @@
+                                    <li>{{ HTML::link('https://cts.vatsim.uk/', 'Central Training System', array("target"=>"_blank")) }}</li>
+                                    <li>{!! link_to_route("networkdata.online", "Who's Online") !!}</li>
+                                    <li>{{ HTML::link('http://www.nats-uk.ead-it.com/public/index.php%3Foption=com_content&task=blogcategory&id=6&Itemid=13.html', 'UK Charts', array("target"=>"_blank")) }}</li>
                                 </ul>
                             </li>
-                            <li class="col-sm-6">
+                        </ul>
+                    </li>
+                </ul>
+
+                <ul class="nav navbar-nav navcustom">
+                    <li class="dropdown dropdown-large">
+                        <a href="#" class="dropdown-toggle" data-toggle="dropdown">Community <b class="caret"></b></a>
+                        <ul class="dropdown-menu dropdown-menu-large row mainmenu_dropdown">
+                            <li class="col-sm-12">
                                 <ul>
+                                    <li class="dropdown-header">Membership</li>
+                                    <li>{!! link_to_route("visiting.landing", "Visit the UK") !!}</li>
+                                    <li>{!! link_to_route("visiting.landing", "Transfer to the UK") !!}</li>
+                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
+                                    <li class="divider"></li>
                                     <li class="dropdown-header">Third-Party Services</li>
                                     <li>{!! link_to_route("teamspeak.new", "TS Registration") !!}</li>
                                     <li>{!! link_to_route("slack.new", "Slack Registration") !!}</li>
-                                    <li class="divider"></li>
-                                    <li class="dropdown-header">Community Features</li>
-                                    <li>{!! link_to_route("mship.notification.list", "View Notifications") !!}</li>
-                                    <li>{!! link_to_route("mship.email", "Email a Member") !!}</li>
-                                    <li>{!! link_to_route("mship.feedback.new", "Controller/Pilot Feedback") !!}</li>
                                 </ul>
                             </li>
                         </ul>
-
                     </li>
                 </ul>
 
-                @if(Auth::check())
-                    <ul class="nav navbar-nav navcustom">
+                @if(Auth::check() || Session::has('auth.vatsim-sso'))
+                    {!! Form::open(['route' => 'logout', 'id' => 'logout-form']) !!}
+                    <ul class="nav navbar-nav navcustom navbar-right">
                         <li class="dropdown dropdown-large">
                             <a href="#" class="dropdown-toggle" data-toggle="dropdown">Account <b class="caret"></b></a>
```

Comment:
I personally think that the name/account would be more useful/noticeable up on the nav bar rather than the breadcrumb bar; what do you think? Thoughts @NFarrington?